### PR TITLE
style refactor

### PR DESCRIPTION
**Pull request by kernyan**
- the script works well, this PR is just some style refactoring for safety
- feel free to edit or ignore
- thanks for open-sourcing the project!

### CODE REVIEW

File: download_node_csv.py [hunk 1]
```diff
@@ -30,78 +34,126 @@ class City(Enum):
...
-[start_nelng, start_nelat, start_swlng, start_swlat] = coordinates
+CityGrids = {
+    City.YORK:        CityGrid(-79.3829, 43.7206, -79.5560, 43.6424),
+    City.WROCLAW:     CityGrid(17.078224311098552, 51.13988879756559,  17.00226573206399,  51.09263199227115 ),
+    City.KRAKOW:      CityGrid(19.979436306324942, 50.08859858611942,  19.90053987336441,  50.034804024531525),
+    City.ROME:        CityGrid(12.519622013860925, 41.91439535724936,  12.443743029831694, 41.85439499689079 ),
+    City.VENICE:      CityGrid(12.357666134722393, 45.451386491714715, 12.316599314442499, 45.42077976598716 ),
+    City.FOLKESTONE:  CityGrid(1.2028963028344322, 51.11176465501126,  1.1199095563368644, 51.05639602006997 ),
+    City.MEAFORD:     CityGrid(-80.49028489574928, 44.754231277837675, -80.94388807383417, 44.44134084860639 ),
+    City.OLD_TORONTO: CityGrid(-79.20, 43.8, -79.556, 43.61)
```
Comment (codereport) [inline download_node_csv.py]: This doesn't duplicate the `else` case on the left. Would need to add:
```py
    City.NORTH_YORK:  CityGrid(-79.20, 43.8, -79.556, 43.61),
    City.EAST_YORK:   CityGrid(-79.20, 43.8, -79.556, 43.61),
    City.ALL_TORONTO: CityGrid(-79.20, 43.8, -79.556, 43.61)
```

**Review by codereport — CHANGES_REQUESTED** [review]
Awesome! This looks fantastic. Only thing is I don't know how to use it now 🙈. Previously I was using a `cookies_headers.py` file described in the `README.md`. I managed to generate a `.json` version using https://www.scrapingbee.com/curl-converter/json/ but it crashes still (once I fixed the problem below) with this:

```
Choice: 11
30 16
Traceback (most recent call last):
  File "/home/cph/city-strides-hacking/./download_node_csv.py", line 156, in <module>
    nodes, cache, cache_file = download_nodes_of_city(city, cookies)
  File "/home/cph/city-strides-hacking/./download_node_csv.py", line 135, in download_nodes_of_city
    response = requests.get(
  File "/home/cph/.local/lib/python3.10/site-packages/requests/api.py", line 73, in get
    return request("get", url, params=params, **kwargs)
  File "/home/cph/.local/lib/python3.10/site-packages/requests/api.py", line 59, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/cph/.local/lib/python3.10/site-packages/requests/sessions.py", line 573, in request
    prep = self.prepare_request(req)
  File "/home/cph/.local/lib/python3.10/site-packages/requests/sessions.py", line 484, in prepare_request
    p.prepare(
  File "/home/cph/.local/lib/python3.10/site-packages/requests/models.py", line 370, in prepare
    self.prepare_cookies(cookies)
  File "/home/cph/.local/lib/python3.10/site-packages/requests/models.py", line 627, in prepare_cookies
    cookie_header = get_cookie_header(self._cookies, self)
  File "/home/cph/.local/lib/python3.10/site-packages/requests/cookies.py", line 147, in get_cookie_header
    jar.add_cookie_header(r)
  File "/usr/lib/python3.10/http/cookiejar.py", line 1375, in add_cookie_header
    attrs = self._cookie_attrs(cookies)
  File "/usr/lib/python3.10/http/cookiejar.py", line 1334, in _cookie_attrs
    self.non_word_re.search(cookie.value) and version > 0):
TypeError: expected string or bytes-like object
```
What did you do for it to work?

**Comment by kernyan** [comment]
> Awesome! This looks fantastic. Only thing is I don't know how to use it now 🙈. Previously I was using a `cookies_headers.py` file described in the `README.md`. I managed to generate a `.json` version using https://www.scrapingbee.com/curl-converter/json/ but it crashes still (once I fixed the problem below) with this:
> 
> ```
> Choice: 11
> 30 16
> Traceback (most recent call last):
>   File "/home/cph/city-strides-hacking/./download_node_csv.py", line 156, in <module>
>     nodes, cache, cache_file = download_nodes_of_city(city, cookies)
>   File "/home/cph/city-strides-hacking/./download_node_csv.py", line 135, in download_nodes_of_city
>     response = requests.get(
>   File "/home/cph/.local/lib/python3.10/site-packages/requests/api.py", line 73, in get
>     return request("get", url, params=params, **kwargs)
>   File "/home/cph/.local/lib/python3.10/site-packages/requests/api.py", line 59, in request
>     return session.request(method=method, url=url, **kwargs)
>   File "/home/cph/.local/lib/python3.10/site-packages/requests/sessions.py", line 573, in request
>     prep = self.prepare_request(req)
>   File "/home/cph/.local/lib/python3.10/site-packages/requests/sessions.py", line 484, in prepare_request
>     p.prepare(
>   File "/home/cph/.local/lib/python3.10/site-packages/requests/models.py", line 370, in prepare
>     self.prepare_cookies(cookies)
>   File "/home/cph/.local/lib/python3.10/site-packages/requests/models.py", line 627, in prepare_cookies
>     cookie_header = get_cookie_header(self._cookies, self)
>   File "/home/cph/.local/lib/python3.10/site-packages/requests/cookies.py", line 147, in get_cookie_header
>     jar.add_cookie_header(r)
>   File "/usr/lib/python3.10/http/cookiejar.py", line 1375, in add_cookie_header
>     attrs = self._cookie_attrs(cookies)
>   File "/usr/lib/python3.10/http/cookiejar.py", line 1334, in _cookie_attrs
>     self.non_word_re.search(cookie.value) and version > 0):
> TypeError: expected string or bytes-like object
> ```
> 
> What did you do for it to work?

oh sorry, ya i didn't explain. so i learned that the default header works so i removed it. the `cookies.json` only contains these two keys like below
```
{
  "remember_user_token": "...",
  "_citystrides_session": "..."
}
```

File: download_node_csv.py (hunk 1 above)
Comment (kernyan) [reply download_node_csv.py]: good catch, fixed!

**Review by codereport — APPROVED** [review]
Awesome! Works great now! Love the `to_csv`. 🙏

**Pull request merged**